pkg/gds/backup_test.go

```diff
@@ -0,0 +1,83 @@
+package gds_test
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/trisacrypto/directory/pkg/gds"
+	"github.com/trisacrypto/directory/pkg/gds/config"
+)
+
+// Test that the backup manager does not create backups if disabled.
+func (s *gdsTestSuite) TestBackupManagerDisabled() {
+	conf := gds.MockConfig()
+	conf.Backup = config.BackupConfig{
+		Enabled:  false,
+		Interval: time.Millisecond,
+		Storage:  "testdata/backup",
+		Keep:     1,
+	}
+	s.SetConfig(conf)
+	defer s.ResetConfig()
+	s.LoadEmptyFixtures()
+	require := s.Require()
+
+	// Start the backup manager
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go s.svc.BackupManager(nil, &wg)
```

Comment:
Above I mentioned that I didn't think that the wait group was necessary to pass to the `BackupManager` because I believe this is simpler in the test case:

```suggestion
	go func() {
		defer wg.Done()
		s.svc.BackupManager(nil)
	}()
```

Now the backup manager doesn't have to worry about the wait group which is only used for tests since the BackupManager should return immediately since it's disabled. 

Note, however, I also mentioned in an early comment that it might be nice to add a stop signal to the `s.Shutdown()` command to gracefully shutdown both the CertManager and the BackupManager -- in that case these would both need a wait group that both routines would be a part of. That wait group would have to be [contextual](https://stackoverflow.com/a/32843750/488917). So it's up to you what you'd like to do - if you think we should add a `Shutdown` step, then we can simply comment that the wait groups are used for tests but will eventually also be used for shutdown. Otherwise, we can go the route suggested above if the wait groups are just for tests.